src/OpenTelemetry.Sampler.AWS/README.md

```diff
@@ -19,16 +19,28 @@ X-Ray remote sampling](https://aws-otel.github.io/docs/getting-started/remote-sa
 
 ```csharp
 using OpenTelemetry;
+using OpenTelemetry.Contrib.Extensions.AWSXRay.Resources;
 using OpenTelemetry.Contrib.Extensions.AWSXRay.Trace;
+using OpenTelemetry.Sampler.AWS;
 using OpenTelemetry.Trace;
 
-var tracerProvider = Sdk.CreateTracerProviderBuilder()
-                        // other configurations
-                        .SetSampler(AWSXRayRemoteSampler.Builder()
-                                                        .SetPollingInterval(TimeSpan.FromSeconds(10))
-                                                        .SetEndpoint("http://localhost:2000")
-                                                        .Build())
-                        .Build();
+var serviceName = "MyServiceName";
+
+var resourceBuilder = ResourceBuilder
+    .CreateDefault()
+    .AddService(serviceName: serviceName)
+    .AddDetector(new AWSEC2ResourceDetector());
+
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .AddSource(serviceName)
+    .SetResourceBuilder(resourceBuilder)
+    .AddConsoleExporter()
+    .AddOtlpExporter()
```

Comment:
Remove exporter from code listing or add some description similar to https://github.com/open-telemetry/opentelemetry-dotnet/blob/a61711d0c37346acfe0f3b096c6ac5a919c9c155/src/OpenTelemetry.Instrumentation.AspNetCore/README.md?plain=1#L52-L54